CHARTER.md

```diff
@@ -0,0 +1,117 @@
+
+   3. TSC projects generally will involve Contributors and Maintainers. The TSC may adopt or modify roles so long as the roles are documented in the MAINTAINERS file. Unless otherwise documented: 
+
+      1. Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project; 
+
+      2. Maintainers are Contributors who have earned the ability to modify (“commit”) source code, documentation or other technical artifacts in a project’s repository; and  
+           
+      3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may be removed by a majority approval of the other existing Maintainers.
+
+   4. Participation in the Project through becoming a Contributor and Maintainer is open to anyone so long as they abide by the terms of this Charter. 
+
+   5. The TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of sub-projects, (2) set requirements for the promotion of Contributors to Maintainer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, and Maintainers, and create new roles, and publicly document any TSC roles, as it sees fit.
+
+   6. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and LF Decentralized Trust, a directed fund of The Linux Foundation.
+
+   7. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+
+      1. coordinating the technical direction of the Project;
+
+      2. approving project or system proposals (including, but not limited to, archiving and changing a sub-project’s scope);
+
+      3. organizing sub-projects and removing sub-projects;
+
+      4. creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+
+      5. appointing representatives to work with other open source or open standards communities;
+
+      6. establishing community norms, workflows, issuing releases, and security issue reporting policies;
+
+      7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
```

Comment:
who is the Series Manager, where is the Series Agreement?

Reply:
This is a common question on many LF docs!

The series agreement refers to the LF legal incorporation documents, and the series manager is the head of LF legal.  What this provision says, in English, is that the project must follow the overall legal rules which bound the LF, and if it does something illegal or something that has the potential to get the LF or maintainers/contributors sued, the LF legal team can step in to get things resolved.